packages/bottender/src/slack/SlackEvent.ts

```diff
@@ -24,6 +24,22 @@ export default class SlackEvent implements Event<SlackRawEvent> {
     return this._rawEvent;
   }
 
+  /**
+   * The timestamp when the event was sent
+   *
+   */
+  get timestamp(): number | undefined {
+    let timestamp: number | undefined;
+    if ('eventTs' in this._rawEvent && this._rawEvent.eventTs !== undefined) {
+      timestamp = parseFloat(this._rawEvent.eventTs);
+    } else if ('ts' in this._rawEvent) {
+      timestamp = parseFloat(this._rawEvent.ts);
+    } else {
+      timestamp = Date.now() / 1000;
```

Comment:
This `Date.now()` should be called in the constructor, so it wouldn't be different between getter calls.

Reply:
And we should rewrite this part to `Date.now()` and `parseFloat(this._rawEvent.ts) * 1000` here, because they're better timestamps that can put into `new Date` directly in JavaScript.